pkg/metadata/aws/apigateway/enable_tracing.go

```diff
@@ -0,0 +1,15 @@
+package apigateway
+
+import "github.com/aquasecurity/cloud-metadata/pkg/metadata"
+
+var EnableTracing = metadata.Metadata{
+	ID:          "AVD-AWS-0003",
+	Title:       "API Gateway must have X-Ray tracing enabled",
```

Comment:
I propose we standardize on an objective statement for the title, and an explanation with a "should" opinionated statement for the description.

```
Title: "API Gateway does not have X-Ray tracing enabled"
```

and

```
"API Gateways should have X-Ray tracing enabled to facilitate end-to-end debugging and analysis of API Gateway requests"
```